src/index.html

```diff
@@ -17,6 +17,90 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="wrapper">
+        <div class="navigation_logo">
+          <a
+            class="logo_link"
+            href="#"
+          >
+            <img
+              class="logo"
+              alt="MOYO company logo"
+              src="images/logo.png"
+            />
+          </a>
+        </div>
+        <nav class="navigation">
+          <ul class="nav_list">
+            <li class="nav_item">
+              <a
+                class="nav_link is-active"
+                href="#"
+              >
+                apple
+              </a>
+            </li>
+            <li class="nav_item">
+              <a
+                class="nav_link"
+                href="#"
+              >
+                samsung
+              </a>
+            </li>
+            <li class="nav_item">
+              <a
+                class="nav_link"
+                href="#"
+              >
+                smartphones
+              </a>
+            </li>
+            <li class="nav_item">
+              <a
+                class="nav_link diff-color"
+                data-qa="hover"
+                href="#"
+              >
+                laptops & computers
+              </a>
+            </li>
+            <li class="nav_item">
+              <a
+                class="nav_link"
+                href="#"
+              >
+                gadgets
+              </a>
+            </li>
+            <li class="nav_item">
+              <a
+                class="nav_link"
+                href="#"
+              >
+                tablets
+              </a>
+            </li>
+            <li class="nav_item">
+              <a
+                class="nav_link"
+                href="#"
```

Comment:
The href attribute of the a tag should not have spaces. It's better to use a meaningful link instead of '#'. If there is no appropriate page, you can use '#photo' to make it more meaningful.